modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/PropertiesFeatureFlagsCheck.java

```diff
@@ -222,6 +239,8 @@ private List<String> _getFeatureFlags(String content, Pattern pattern) {
 		return featureFlags;
 	}
 
+	private static final Pattern _deprecationFeatureFlagPattern =
+		Pattern.compile("feature\\.flag[.=]([A-Z]+-\\d+)\\.type=deprecation");
```

Comment:
I know this was based off the Regex below, but in both cases I don't think we need `[.=]`. I think that should just be able to be `\\.` (an escaped period). @drewbrokke is there any reason why we still need this extra complexity in the Regex? Is this a hold-over from some old code?

Reply:
Good catch @ethib137, this was left over from the regex immediately below this one. We can get rid of the `=` and just use `\\.`. Thanks!

Reply:
@ethib137 @drewbrokke  Great, changing the regexes to:

`feature\\.flag\\.([A-Z]+-\\d+)\\.type=deprecation`

and

`feature\\.flag\\.([A-Z]+-\\d+)`

Reply:
Just double checking @drewbrokke is it okay to make that change to both instances of Regex?

Reply:
@thiago-buarqque no, please only change the new regex. The other one has the equals sign on purpose and handles more use-cases than just the properties.

Reply:
Okay then, thanks